Jenkinsfile.pr

```diff
@@ -133,15 +133,11 @@ node('docker') {
     try {
```

Comment:
I can't find any reference to this file anywhere else. We should remove it, can easily put it back if something breaks.